source/configuring_domain/05_configuring_domain/regional_settings.rst

```diff
@@ -18,3 +18,19 @@ To change the timezone of a domain, go to Configuration > Regional settings. In
    :class: note
 
    It is also possible to directly write a city or zone in the textbox. A selection of matching existing timezones will appear.
+
+
+Datasets default location
+-------------------------
+
+The :ref:`geographic coverage metadata <geographic-coverage-metadata>` indicates the location of the data contained in the dataset. Although the metadata can be set individually for each dataset, it is possible to configure the default location(s) that will be automatically set as geographic coverage metadata for all datasets created afterward.
+
+1. In the Default location textbox, write the name of the chosen location.
+2. A drop-down selection of existing locations matching the name that was previously typed appears. Click on the right location.
+3. (optional) Repeat steps 1 and 2 of this procedure to add other default locations.
```

Comment:
Maybe add, as a note, the cases when having several default locations is a good idea